obspy/taup/tau.py

```diff
@@ -379,6 +391,167 @@ def get_ray_paths(self, source_depth_in_km, distance_in_degree=None,
         return Arrivals(sorted(rp.arrivals, key=lambda x: x.time),
                         model=self.model)
 
+    def get_travel_times_geo(self, source_depth_in_km, source_latitude_in_deg,
+                             source_longitude_in_deg, receiver_latitude_in_deg,
+                             receiver_longitude_in_deg, phase_list=("ttall",)):
+        """
+        Return travel times of every given phase given geographical data.
+
+        :param source_depth_in_km: Source depth in km
+        :type source_depth_in_km: float
+        :param source_latitude_in_deg: Source latitude in degrees
+        :type source_latitude_in_deg: float
+        :param source_longitude_in_deg: Source longitude in degrees
+        :type source_longitude_in_deg: float
+        :param receiver_latitude_in_deg: Receiver latitude in degrees
+        :type receiver_latitude_in_deg: float
+        :param receiver_longitude_in_deg: Receiver longitude in degrees
+        :type receiver_longitude_in_deg: float
+        :param phase_list: List of phases for which travel times should be
+            calculated. If this is empty, all phases will be used.
+        :type phase_list: list of str
+
+        :return: List of ``Arrival`` objects, each of which has the time,
+            corresponding phase name, ray parameter, takeoff angle, etc. as
+            attributes.
+        :rtype: :class:`Arrivals`
+
+        Note that the conversion from source and receiver latitudes and
+        longitudes to epicentral distances respects the model's flattening
+        parameter, so this calculation can be performed for a ellipsoidal
+        or spherical Earth. However, the actual traveltime and raypath
+        calculations are performed on a spherical Earth. Ellipticity
+        corrections (e.g. Dziewonski and Gilbert 1976, Geophys. J. R. astr.
+        Soc. 44:7-17) are not made.
```

Comment:
Can you please convert that to an actual reference? For ObsPy this means to add a bib file here: https://github.com/obspy/obspy/tree/master/misc/docs/source/bibliography

and then you can refer to it in the docstring, e.g. like this: `[Dziewonski1976]_`

The documentation then contains nice clickable links and this overview: http://docs.obspy.org/citations.html

Thanks :-)